native/c/zbase64.h

```diff
@@ -276,6 +307,26 @@ inline std::vector<char> decode(const char *input, size_t input_len)
   return output;
 }
 
+inline std::vector<char> decode(const char *input, size_t input_len, std::vector<char> *left_over)
+{
+  // Prepend leftover data to input string
+  const char *combined_input = input;
+  std::vector<char> temp_combined;
+  if (left_over != nullptr && !left_over->empty())
+  {
+    input_len = load_leftover(input, input_len, *left_over, temp_combined);
+    combined_input = &temp_combined[0];
+  }
+  else if (input_len == 0)
+  {
+    return std::vector<char>();
```

Comment:
You could just return `temp_combined` here

```suggestion
    return temp_combined;
```